SCClassLibrary/JITLib/ProxySpace/NodeMap.sc

```diff
@@ -93,10 +93,10 @@ ProxyNodeMap : NodeMap {
 
 	put { |key, item|
 		if(proxy.notNil) {
-			if(this.at(key).isKindOf(NodeProxy)) {
+			if(this.at(key).proxyNeedsWakeUp) {
```

Comment:
The method name `proxyNeedsWakeUp` is too specific. We add children for at least two different reasons: waking up parents, but also rebuilding when the channels have changed. And also `proxyNeedsWakeUp` reads a bit as a question "does this proxy need wake up?" – which is misleading, because it is true for every proxy and only false for other objects.

Let's see what is the best solution, below.